pkg/plugins/golang/v2/scaffolds/internal/templates/makefile.go

```diff
@@ -154,35 +154,31 @@ deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in
 undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
 	$(KUSTOMIZE) build config/default | kubectl delete --ignore-not-found=$(ignore-not-found) -f -
 
-.PHONY: controller-gen
-controller-gen: ## Download controller-gen locally if necessary.
-ifeq (, $(shell which controller-gen))
-	@{ \
-	set -e ;\
-	CONTROLLER_GEN_TMP_DIR=$$(mktemp -d) ;\
-	cd $$CONTROLLER_GEN_TMP_DIR ;\
-	go mod init tmp ;\
```

Comment:
could we not try to just replace go get by install? 
This shows that we are trying to do a refac in the file and it will make it harder for the users to upgrade the changes

Reply:
No, explanation in PR description.

Reply:
Hi @ryantking please see: https://github.com/kubernetes-sigs/kubebuilder/issues/2480